### ISSUE TITLE
get_rnaseq doesn't have region parameter

### ISSUE THREAD

**Issue by cancenik** [ribosomeprofiling/riboR_alpha#15]
Would be good to include the region parameter for get_rnaseq

**Comment by mjgeng** [comment]
I did not get to this today, but I should have it done by tomorrow!

**Comment by mjgeng** [comment]
I added the 'region' parameter. Additionally, in both the get_region_counts and the get_rna_seq, I fixed the ordering of the columns in the returned data.table. This means that the user can specify a character vector of regions in any order, and when tidy = FALSE, the returned data.table will always output columns in the order of UTR5, UTR5J, CDS, UTR3J, and UTR3. 

`ingolia <- ribo("hek293_ingolia.ribo")`
` rna_seq2 <- get_rnaseq(ingolia, regions = c("UTR5", "CDS", "UTR3"), tidy = FALSE)`
 `rna_seq1 <- get_rnaseq(ingolia, regions = c("CDS", "uTR5", "utR3"), tidy = FALSE)`
 `all(rna_seq1 == rna_seq2)`
  `   [1] TRUE`
